src/build/app/mod.rs

```diff
@@ -667,6 +668,27 @@ impl<'help> App<'help> {
         self
     }
 
+    /// Sets a string of the license to be displayed when displaying help information.
+    ///
+    /// **Pro-tip:** Use `clap`s convenience macro [`crate_license!`] to automatically set your
+    /// application's license to the same thing as your crate at compile time. See the
+    /// [`examples/`] directory for more information
```

Comment:
```suggestion
    /// [`examples`] directory for more information
```

Reply:
All other occurrences of `[examples/]` in the same file have a trailing forward slash. If you think it's better to remove it, I think it should be done all at once in a separate commit.